code/game/gamemodes/clock_cult/clock_cult.dm

```diff
@@ -112,27 +112,35 @@ Credit where due:
 		restricted_jobs += protected_jobs
 	if(config.protect_assistant_from_antagonist)
 		restricted_jobs += "Assistant"
-	var/starter_servants = 3 //Guaranteed three servants
+	var/starter_servants = 1 //Guaranteed three servants
 	var/number_players = num_players()
 	roundstart_player_count = number_players
 	if(number_players > 30) //plus one servant for every additional 15 players
 		number_players -= 30
 		starter_servants += round(number_players/15)
+	time_to_prepare = max(20, min(number_players / 2, 30))
+	GLOB.clockwork_potential = 25 + (number_players * 2)
 	while(starter_servants)
 		var/datum/mind/servant = pick(antag_candidates)
 		servants_to_serve += servant
 		antag_candidates -= servant
 		modePlayer += servant
+		servant.assigned_role = "Servant of Ratvar"
 		servant.special_role = "Servant of Ratvar"
-		servant.restricted_roles = restricted_jobs
 		starter_servants--
+	addtimer(CALLBACK(src, .proc/cry_havoc), 100)
+	//addtimer(CALLBACK(src, .proc/cry_havoc), time_to_prepare * 600) //600 deciseconds in a minute, so multiply the number of minutes to prepare times 600
 	return 1
 
 /datum/game_mode/clockwork_cult/post_setup()
 	for(var/S in servants_to_serve)
 		var/datum/mind/servant = S
 		log_game("[servant.key] was made an initial servant of Ratvar")
 		var/mob/living/L = servant.current
+		if(GLOB.servant_start.len)
```

Comment:
```LAZYLEN()```, surely? (the ```LAZYREMOVE()``` below implies this list can be null, if that is not the case then use a real list remove below)